meson.build

```diff
@@ -5,16 +5,26 @@ project('vkd3d', ['c'], version : '1.1', meson_version : '>= 0.51', default_opti
 cpu_family = target_machine.cpu_family()
 
 vkd3d_compiler = meson.get_compiler('c')
+vkd3d_msvc     = vkd3d_compiler.get_id() == 'msvc'
 vkd3d_c_std    = 'c99'
 vkd3d_platform = target_machine.system()
 
 enable_tests             = get_option('enable_tests')
 enable_extras            = get_option('enable_extras')
+enable_standalone_d3d12  = get_option('enable_standalone_d3d12')
+
+if vkd3d_platform != 'windows' and enable_standalone_d3d12
```

Comment:
I generally had problems with this on MinGW. vkd3d_platform is detected as mingw64_nt or something like that in msys2.

Reply:
You **need** to use the cross-file, not just set CC=mingw-w64-gcc, etc.

Reply:
Even when building on native Windows? From Linux cross-compiling, sure.

Reply:
You should still use the cross file, it *should* work -- also I'd highly recommend using WSL over msys2 for building like this.